homework/index.js

```diff
@@ -30,13 +30,118 @@
     return elem;
   }
 
+  function setInnerText(repository, nthChild, text) {
+    document.querySelector('tbody').childNodes[nthChild].lastChild.innerText = text;
+  }
+
+  function presentRepositories(leftHandColumn, repository) {
+    const table = createAndAppend('table', leftHandColumn);
+    const tBody = createAndAppend('tbody', table);
+    const headsOfDetails = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < 4; i++) {
```

Comment:
This for-loop doesn't seem the best way to do it. Firstly because you are hard-coding the index length as 4 instead of using `headsOfDetails.length`. Or you can try using the `forEach` method. Also, you need not necessarily use the inner for loop as you can simply create two `<td>` elements inside the `forEach`.

So, after creating the `table` and `tbody` tags, you can execute a forEach where you create two `td` elements and assign the `headsOfDetails` to the first ones. After that, you can assign the second ones with the actual information. Does that make sense?

Reply:
Similar to what you are doing in the `renderContributors` function

Reply:
I applied forEach for the `<tr>` tags too. Yeah, that was bad conduct of coding. Thanks :pray: